modules/ROOT/pages/connected-apps-overview.adoc

```diff
@@ -0,0 +1,22 @@
+= Connected Apps
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can connect applications to Anypoint Platform without granting them unrestricted access to data or having to share sensitive credentials.
+When you grant an application access, you have direct control over its data access, usage, and credentials.
+You can audit the use of data by connected apps, and you can also revoke access at any time.
+
+== Authentication
+
+Using OAuth 2.0 and OpenID, you can authorize applications to access resources using tokens or Single Sign-On.
+Additionally, you can share account data with third-party applications.
+For more information on which users can grant access to applications using Anypoint Platform, see Connected Apps Permissions by User.
```

Comment:
Are you adding a link to that topic in See Also?

Reply:
I can.